.github/workflows/test.yaml

```diff
@@ -122,6 +122,11 @@ jobs:
           # Just Python 3.9 with default settings.
           - python-version: '3.9'
 
+          # newest macOS and newest Python supported version.
+          - python-version: '3.10'
+            runs-on: 'macos-12'
+            job-name: 'macos-default-tests'
```

Comment:
I think it would be good to keep testing an older version of Python on macOS.  On Azure it looks like we tested 3.7 - 3.10.  Perhaps we can at least test 3.7 _and_ 3.10 (skipping the intermediates) on GA.